airflow/opt/entrypoint-with-secret-files.sh

```diff
@@ -23,7 +23,7 @@ if [ -n "$USE_MOCKS" ]; then
 fi
 
 
-cp /opt/airflow/.airflowignore $(airflow config get-value core dags_folder)/.airflowignore
+cp /opt/airflow/.airflowignore $(airflow config get-value core dags_folder)/
```

Comment:
When running `pdoc`, I was getting an exception on this `cp` command, where it assumed the `$(airflow config get-value core dags_folder)/.airflowignore` was a non-existent directory. Update the target to avoid the error.